koku/masu/processor/gcp/gcp_report_processor.py

```diff
@@ -248,16 +248,16 @@ def _get_or_create_gcp_project(self, row, report_db_accessor):
         data = self._get_data_for_table(row, table_name._meta.db_table)
         data = report_db_accessor.clean_data(data, table_name._meta.db_table)
 
-        key = (data["account_id"], data["project_id"])
+        key = (data["account_id"], data["project_id"], data["project_name"])
         if key in self.processed_report.projects:
             return self.processed_report.projects[key]
 
         if key in self.existing_projects_map:
             return self.existing_projects_map[key]
 
         with transaction.atomic():
-            project_id = report_db_accessor.insert_on_conflict_do_nothing(
-                table_name, data, conflict_columns=["project_id"]
+            project_id = report_db_accessor.insert_on_conflict_do_update(
```

Comment:
@adberglund does anything need to be done for trino?

Reply:
@dccurtis I actually had the same concerns when I was working on this. I chatted with @adberglund to make sure we don't need to do the same change in the parquet processor. From my understanding, the parquet processor doesn't use the `reporting_gcpproject` table, it just populates the data needed in the summary table. So, instead of updating the row in the `reporting_gcpproject` table in the parquet processor, the new row would just be inserted into the daily summary table with the correct value.

Reply:
ok great!  That was what I suspected but I'm glad you confirmed it. 👍